projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -45,24 +60,85 @@
   &:nth-child(2n) {
     background: $theme-light-bg-active;
   }
-
-  &:last-child .go-table__td {
-    border-bottom: 1px solid $theme-light-border;
-  }
 }
 
 .go-table__td {
   padding: 0.5rem 1rem;
+}
+
+.go-table-toolbar {
+  align-items: center;
+  border: 1px solid $theme-light-border;
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5rem;
+}
+
+.go-table-toolbar--bottom {
+  border-radius: 0 0 $global-radius $global-radius;
+  border-top: 0;
+}
+
+.go-table-page-controls {
+  align-items: center;
+  display: flex;
+}
 
-  &:first-child {
-    border-left: 1px solid $theme-light-border;
+.go-table-page-controls__icon {
+  align-items: center;
+  border-radius: 50%;
+  cursor: pointer;
+  display: flex;
+  font-size: 1.25rem;
+  margin: 0 0.25rem;
+  padding: 0.5rem;
+  user-select: none;
+  @include transition(all);
+
+  &:hover {
+    background: $theme-light-bg-hover;
   }
+}
 
-  &:last-child {
-    border-right: 1px solid $theme-light-border;  
+.go-table-page-controls__icon--disabled {
```

Comment:
This will be able to be refactored out with the updates we are making to the go-icon.